README.md

```diff
@@ -473,6 +473,8 @@ The following environment variables can be used to configure reporting to Cloudw
 
 * **CLOUDWATCH_ON** - `default false` reports metricsRegistry to cloudwatch, TO_REGISTRY will need to be switched on!
 * **CLOUDWATCH_NAME** - `default 'remora'` name to appear on cloudwatch
+* **WHITELIST** - `default ''` metric names to whitelist. The whitelist will be compared with metric names and will allow based on "equality" or "endswith". I.E "gauge.DATA_SAMPLE.lag" will be whitelisted if the environment variable is "DATA_SAMPLE.lag
```

Comment:
In general, we're moving away from using whitelist as a term in our code. It would be better to call this what it actually is - i.e. `METRICS_FILTER` or `CLOUDWATCH_METRICS_FILTER`

Reply:
Instead of a space delimited list of expression for 'endsWith' I think allowing users to provide a Regex expression here would allow for more configuration options

Reply:
Changed to checking regex match rather than string equality or ends with and renamed it cloudwatch_metrics_filter